pipeline_tools/get_upload_urn.py

```diff
@@ -58,14 +61,16 @@ def get_staging_urn(envelope_js):
 
```

Comment:
Should we also update the function `get_staging_urn()` and its docstring here, since the format is changed now?